controllers/helm.go

```diff
@@ -189,16 +189,29 @@ func updateInfraChartsFromInstall(ctx context.Context, in *v1beta1.Installation,
 			if clusterConfig.Spec != nil && clusterConfig.Spec.Network != nil {
 				serviceCIDR = clusterConfig.Spec.Network.ServiceCIDR
 			}
+			if in.Spec.Network != nil {
+				serviceCIDR = in.Spec.Network.ServiceCIDR
```

Comment:
might be a good idea to move the logic for determining service CIDRs to a util function

Reply:
yeah since it's in a few places
will do